utils/timings.ts

```diff
@@ -5,7 +5,21 @@ export interface Timing {
   end?: number;
 }
 
-export function createServerTimings() {
+export interface ServerTimingsReturn {
```

Comment:
the word `return` is redundant as everything can be "returned" from somewhere
```suggestion
export interface ServerTimingsBuilder {
```